Issue/13

OlafvanDijk opened the pull request:
- issue 13 en 12
- 1/2 kans dat jason achterstevoren wordt uitgelezen
- gebruiker kan aantal auto's invoeren die hij/zij in wil spawnen
- BUG als 1 auto klaar is met zijn route zullen alle andere auto's die dezelfde route volgden stoppen met rijden.

OlafvanDijk committed:
issue 13 en 12

- 1/2 kans dat json achterstevoren uitgelezen wordt
- input veld om een aantal auto's in te laten spawnen
- BUG als 1 auto klaar is met zijn rit zullen alle andere auto's die dezelfde rit volgde stoppen met rijden.

OlafvanDijk committed:
Personal info got logged

BakkerTom commented on models/geolocation-simulator.js:
```diff
@@ -98,7 +98,15 @@ module.exports = function(params) {
         if (_done) {
+            if(_coords[_coords.length-1].latitude == _current.coords.latitude)
+            {
+            //console.log("Point to reach= Lat:" + _coords[_coords.length -1].latitude + " Long:" + _coords[_coords.length -1].longitude + " Current= Lat:" + _current.coords.latitude + " Long" + _current.coords.longitude);
```
Misschien deze even weghalen

BakkerTom commented on public/javascripts/scripts.js:
```diff
@@ -81,7 +82,10 @@ socket.on('car update', (car) => {
 
 addCarBtn.addEventListener("click", () => {
-  fetch("/cars", { method: "POST" }).then(() => {});
+  for (let index = 0; index < amountOfCars.value; index++) {
```
Dit verklaart volgens mij de memory overflow... Je kunt beter een request maken:

```javascript
fetch(`/cars?amountOfCars=${amountOfCars.value}`, { method: "POST" }).then(() => {});
```

En dan halen we deze parameter op op de server en starten `simulateCar()` meerdere malen.

BakkerTom reviewed (commented):
Gaat de goede kant op! Heb nog wel een paar dingen die ik graag zie... Voornamelijk de for loop in het front-end. Dit zie ik liever in het backend terug.

OlafvanDijk committed:
For is now in the backend

OlafvanDijk committed:
Bug fixes

- You can now spawn more than 2 cars at a time
- When a car is done it will only delete itself now
